src/servers/src/grpc/prom_query_gateway.rs

```diff
@@ -62,6 +63,13 @@ impl PrometheusGateway for PrometheusGatewayService {
         };
 
         let query_context = create_query_context(inner.header.as_ref());
+        let _timer = timer!(
+            crate::metrics::METRIC_SERVER_GRPC_PROM_REQUEST_TIMER,
+            &[(
+                crate::metrics::METRIC_DB_LABEL,
+                &query_context.get_db_string()
```

Comment:
I think it might be better to separate the catalog and schema into two lables.

Reply:
We don't expose catalog to open source user so for here db is just schema. We might not need to expose catalog again in metrics.